ibc-clients/cw-context/src/context/mod.rs

```diff
@@ -195,26 +195,22 @@ impl<'a, C: ClientType<'a>> Context<'a, C> {
 
         let iterator = self
             .storage_ref()
-            .range(Some(&start_key), None, Order::Ascending);
+            .range(Some(&start_key), None, Order::Ascending)
+            .filter(|(key, _)| key.starts_with(ITERATE_CONSENSUS_STATE_PREFIX.as_bytes()));
 
```

Comment:
A small nit:
Let's have the `let start_key` above as `ITERATE_CONSENSUS_STATE_PREFIX.as_bytes()` and use it here for the `key.starts_with(start_key)` as well.

Reply:
Ah yes. I missed this :sweat_smile:

Reply:
makes sense ! but I refactored the `.range` calls with `cw_storage_plus::Map`. let's comment there.